packages/dds/tree/src/simple-tree/tree.ts

```diff
@@ -23,28 +23,30 @@ import {
  */
 export interface ITree extends IChannel {
 	/**
-	 * Returns a tree that is known to be compatible with the provided schema. The returned tree exposes an API that is schema-aware.
+	 * Returns a {@link TreeView} using the provided schema.
+	 * If the tree's stored schema is compatible, this will provide a schema aware API for accessing the tree's content.
+	 * If the provided schema is incompatible with the stored data, the view will instead expose an error indicating the incompatibility.
 	 *
 	 * @remarks
 	 * If the tree is uninitialized (has no schema and no content), the tree is initialized with the `initialTree` and
 	 * view `schema` in the provided `config`.
 	 *
 	 * The tree (now known to have been initialized) has its stored schema checked against the provided view schema.
 	 *
-	 * If the schemas are compatible (including updating the stored schema if permitted via `allowedSchemaModifications`),
-	 * a {@link TreeView} is returned, with a schema-aware API based on the provided view schema.
+	 * If the schemas are compatible the returned {@link TreeView} with expose the root with a schema-aware API based on the provided view schema.
 	 *
-	 * If the schemas are not compatible, an exception is thrown.
+	 * If the schemas are not compatible, the view will indicate the error.
+	 *
+	 * Note that other clients can modify the document at any time, causing the view to enter or leave this error state: see {@link TreeView} for how to handle invalidation in these cases.
```

Comment:
Not sure that this link works, but if it does it seems preferable:


```suggestion
	 * Note that other clients can modify the document at any time, causing the view to enter or leave this error state: see {@link TreeView.events} for how to handle invalidation in these cases.
```